### PR TITLE
Chemical Wiki Descriptions, Recipe Tweaks & Effect Tweaks (2)

### PR DESCRIPTION
**Pull request by Alekshhh**
## About the PR
o Changes almost all wordings in the chemical wiki section to have up-to-date, consistent and relevant information.
o Tweaked chemical recipes, namely:

- All plasma catalysts were reduced from 5u to 1u
- Epinephrine no longer requires diethylamine, creates 5u epinephrine
- Dexalin requires 2u hydrogen,1u oxygen, 1u plasma catalyst, creates 3u dexalin
- Lexorin requires 1u heartbreaker toxin, 1u fluorine, 1u plasma, creates 3u lexorin
- Ultravasculine requires 1u dylovene, 1u sodium, 1u plasma catalyst
- Sigynate requires 1u iodine, 1u kelotane, 1u sugar, creates 3u sigynate
- Phenol requires 1u oxygen, 1u hydrogen, 1u carbon
- Diphenymethylamine requires 1u ethyloxyephedrine, 1u phenol, 1u sodium carbonate, makes 3u diphenylmethylamine
- Ammonia reaction creates 3u instead of 4u
- Sodium carbonate no longer requires ammonia, creates 3u sodium carbonate

o Tweaked chemical effects, namely:

- Nocturine no longer causes poison damage on overdose (60u to deal 2 poison per second are you real)
- Sigynate cures 2 caustic a second instead of 6

**Media**

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl: Alekshhh
- tweak: The chemical page of the wiki has been heavily reworded, and a few chemicals had their recipes tweaked and effects changed a little

### CODE REVIEW

File: Resources/Locale/en-US/reagents/meta/toxins.ftl [hunk 1]
```diff
@@ -1,50 +1,50 @@
...
-reagent-desc-fluorosulfuric-acid = An extremely corrosive chemical substance.
+reagent-desc-fluorosulfuric-acid = A highly corrosive chemical, capable of leaving quite a mark on your skin.
 
 reagent-name-sulfuric-acid = sulfuric acid
-reagent-desc-sulfuric-acid = A highly corrosive, oily, colorless liquid.
+reagent-desc-sulfuric-acid = A corrosive chemical. Keep away from your face.
 
 reagent-name-unstable-mutagen = unstable mutagen
-reagent-desc-unstable-mutagen = Causes mutations when injected into living people or plants. High doses may be lethal, especially in humans.
+reagent-desc-unstable-mutagen = A concoction of irradiated chemicals that are prone to rapid mutation within plants - dionas being the exception. Use with caution.
```
Comment (AJCM-git) [inline Resources/Locale/en-US/reagents/meta/toxins.ftl:23]: weird wording `chemicals that are prone to rapid mutation within plants`

**Review by AJCM-git — CHANGES_REQUESTED** [review]

**Comment by metalgearsloth** [comment]
@Alekshhh are you coming back to this.

**Comment by AJCM-git** [comment]
re-open when you come back to this

**Comment by metalgearsloth** [comment]
> re-open when you come back to this

I agreed to work on it in exchange for laser raptor.

**Comment by Aisu9** [comment]
was it needed to make chemistry no longer needing Botanist help to heal caustic damage efficiently?
Derma now trivialise burn healing
Hyro and Dylo both heal poison and radiation damage
Not a single medicine alleviate bloodloss damage
Epinephrine only work on critical patient
And for the best, Occuline deals eye damage instead of healing eye damage

**Comment by Aisu9** [comment]
Doxarubixadone is still a waste of cryoxadone with how easy you can synthetise phalanximine

**Comment by LankLTE** [comment]
I don’t see why this needs to be made? Updating guidebook descriptions is fine, but then it also makes a bunch of seemingly random Chem changes with no explanation as to why. Sigynate in particular was super underused as-is and I don’t see why Dexalin needs to be harder to make.

**Comment by hord-brayden** [comment]
This wasn't a great one, seemed pretty random to alter reagents to make some recipes easier and others harder.

I'm unsure if this was intentional, but lexorin was possible to make (albeit very diluted) without a chemmaster4000 or Chen dispenser by anyone given they find a few common ingredients- now with Florine, lexorin is now effectively ONLY for chemists.

**Pull request merged**